packages/components/pagination/src/pagination.ts

```diff
@@ -323,14 +324,18 @@ export default defineComponent({
         }
       })
 
-      addClass(rootChildren[0], 'is-first')
-      addClass(rootChildren[rootChildren.length - 1], 'is-last')
+      if (haveRightWrapper && rightWrapperChildren.length > 0) {
+        rootChildren.unshift(rightWrapperRoot)
+      }
```

Comment:
Why add this?

Reply:
😢Left over from previous conflict resolution, now fixed！